injuries/tsd.json

```diff
@@ -0,0 +1,8 @@
+{
+  "version": "v4",
+  "repo": "borisyankov/DefinitelyTyped",
+  "ref": "master",
+  "path": "src/tsd",
+  "bundle": "src/tsd/tsd.d.ts",
+  "installed": {}
```

Comment:
save the dependencies.  This should have react, cu-events, cu-components, cu-core at least. !fix